backend/app.py

```diff
@@ -1,32 +1,35 @@
-from flask import Flask, request, jsonify, send_from_directory, url_for
+# Good resource to build this out:
+# https://blog.miguelgrinberg.com/post/how-to-create-a-react--flask-project
+
+import util
+import notifications
+from models import Ask, Note, NoteTypes
+from errors import bad_request
+from flask import Flask, request, jsonify, url_for, send_from_directory
```

Comment:
should reorganize imports to match PEP:
https://www.python.org/dev/peps/pep-0008/#imports
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.

Reply:
fixed in latest change